ci_framework/roles/set_openstack_containers/templates/update_env_vars.sh.j2

```diff
@@ -0,0 +1,35 @@
+---
+{% set containers_dict = {} %}
+{% set _container_registry = cifmw_set_openstack_containers_registry + '/' + cifmw_set_openstack_containers_namespace %}
+{% for container in containers_env_list.stdout_lines if '=' in container %}
+{% set container_data = container.split('=') %}
+{% set _container_env = container_data[0] %}
+{% set _container = container_data[1] %}
+{% if _container.endswith('current-podified') %}
+{% set _container_data = _container.split('/')[-1] %}
+{% set _container_name = _container_data.split(':')[0] %}
+{% set _container_address = _container_registry + '/' + _container_name + ':' + cifmw_set_openstack_containers_tag %}
+{% set _ = containers_dict.update({_container_env: _container_address}) %}
+{% elif cifmw_set_openstack_containers_overrides.items()|length > 0 %}
+{% if _container_env in cifmw_set_openstack_containers_overrides.items() %}
+{% set _ = containers_dict.update({_container_env: cifmw_set_openstack_containers_overrides[_container_env]}) %}
+{% endif %}
+{% endif %}
+{% endfor %}
```

Comment:
In order to avoid a ton of blank line, I'd use `-%}` instead of the current `%}`